### PR TITLE
Create cert directory for local setup

### PR DESCRIPTION
**Pull request by jay-elizaga-dev**
Solution to issue #8465

### CODE REVIEW

File: examples/compose/local/setup.sh [hunk 1]
```diff
@@ -44,8 +46,13 @@ fi
...
 
-echo "Resetting docker-compose.yml, configuration.yml and users_database.yml"
-sudo git checkout -- docker-compose.yml authelia/configuration.yml authelia/users_database.yml
+echo "Resetting docker-compose.yml, (authelia) configuration.yml and users_database.yml, (traefik) certificates.yml"
+sudo git checkout -- docker-compose.yml authelia/configuration.yml authelia/users_database.yml traefik/certificates.yml
+
+if [[ ! -d "$DIR_TRAEFIKCERT" ]]; then 
+  echo "$DIR_TRAEFIKCERT does not exist.  Creating empty directory: $DIR_TRAEFIKCERT."
+  mkdir "$DIR_TRAEFIKCERT" ; 
+fi
```
[the remark this answers is not in the document]
Comment (jay-elizaga-dev) [reply examples/compose/local/setup.sh:55]: I mean.... this is such an edge case. I can add it if the owners of the repo would like me to.

**Comment by jay-elizaga-dev** [comment]
I would be happy to change the merge branch to an appropriate branch based on the owner's build-release schedule.  Thank you

**Comment by jay-elizaga-dev** [comment]
This is a Podman-specific issue.  Closing

**Comment by james-d-elliott** [comment]
I suspect this may even be a docker-in-podman issue rather than podman itself, I'll set it up on a VM later to confirm.

**Comment by jay-elizaga-dev** [comment]
Sure.  I will add this link for posterity-sake: https://github.com/containers/podman/issues/6234

In short, the current developers of podman chose not to follow docker's convention of creating a nonexistent folder/file if it's mounted.

**Comment by james-d-elliott** [comment]
Ah that's interesting. Maybe we should just add the folder plus a `.gitkeep` then, strange that podman adds a `docker` alias when it doesn't implement the same API.

**Comment by jay-elizaga-dev** [comment]
I agree with the idea and substitute .gitkeep with a .gitignore that ignores all possible certificate files to avoid security flaws

for example 
```
# filename: .gitignore

# contents 
*.pem
*.any.credential.file.*
```

Also I suppose it would probably be too much to make a new issue/pull request out of  it.  Whicher works :)

**Pull request closed without merging**